picamera2/previews/drm_preview.py

```diff
@@ -150,23 +150,24 @@ def render_drm(self, picam2, completed_request):
 
             self.plane = self.resman.reserve_overlay_plane(self.crtc, format=fmt)
             if self.plane is None:
-                raise RuntimeError("Failed to reserve DRM plane")
+                self.plane = self.resman.reserve_plane(self.crtc, type=pykms.PlaneType.Primary, format=fmt)
+                if self.plane is None:
+                    raise RuntimeError("Failed to reserve DRM plane")
             drm_rotation = 1
             if self.transform.hflip:
                 drm_rotation |= 16
             if self.transform.vflip:
                 drm_rotation |= 32
-            self.plane.set_prop("rotation", drm_rotation)
+            #self.plane.set_prop("rotation", drm_rotation)
```

Comment:
Does removing this break display transforms? What happens if we call it and it isn't supported?